engine/src/main/java/org/hibernate/validator/spi/properties/GetterPropertySelectionStrategy.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.spi.properties;
+
+import java.util.Set;
+
+import org.hibernate.validator.Incubating;
+import org.hibernate.validator.internal.properties.DefaultGetterPropertySelectionStrategy;
+
+/**
+ * Used to define the strategy to detect the getter of a bean. The default
+ * implementation ({@link DefaultGetterPropertySelectionStrategy}) uses the JavaBeans
```

Comment:
I wouldn't link to an implementation class, also if it's just JavaDoc.

Reply:
OK, done.